web/modules/custom/hoeringsportal_audit_log/src/Routing/RoutingChangeListener.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace Drupal\hoeringsportal_audit_log\Routing;
```

Comment:
Event subscribers are usually put in the `EventSubscriber` sub-namespace, i.e. `Drupal\hoeringsportal_audit_log\EventSubscriber` in the current case.